Samples/TrendingMovies/TrendingMovies/Utilities/Tracer.swift

```diff
@@ -28,7 +28,7 @@ extension Tracer {
         }
 
         SentrySDK.start { options in
-            options.dsn = "https://a92d50327ac74b8b9aa4ea80eccfb267@o447951.ingest.sentry.io/5428557"
+            options.dsn = "https://fff20ae0c1d141fda99ba8bdedd0e9cd@o447951.ingest.sentry.io/6509889"
```

Comment:
`m`: Can't we share the same project? It would be nice to see the data in our existing test project.

Reply:
I think we should keep this data separate, but as a separate work item, I would like to upload the iOS-Swift app's symbols for the benchmark jobs, so we can see those symbols in the profiling data that gets uploaded to that project as part of the benchmark test runs.

But, I would be open to considering why it would be beneficial to mix the data from separate apps...

Reply:
Ah yes, it's a separate app. Ignore me.